docs/reference/remote-authenticator.md

```diff
@@ -1,7 +1,7 @@
 ---
 layout: default
 title: Remote Authenticator
-description: This section covers lakeFS's remote authenticator architecture for integrating with existing security infrastructure
+description: Integrate lakeFS with your existing security infrastructure
```

Comment:
This is less useful than what you're replacing (you've omitted the remote authentication bit). Please reword.

Reply:
IMHO `lakeFS'` is not the end of the world - is it really worth this set of edits? There's lots of other doc work to do if we have the spare bandwidth. Just my 0.02p ;)

Reply:
About this concrete comment: a "remote authenticator" is not really a thing - it's something that we invented, and hence the title of the page. Then the description would explain what it does for you. Sometimes less is more:
<img width="787" alt="image" src="https://user-images.githubusercontent.com/3149885/233336234-bfdf5415-8f8e-4342-b9c6-9b61b4dac1b2.png">(illustration)

However, since there's no consensus, I'm rewording to be more with the original spirit of the sentence.